.gitignore

```diff
@@ -7,16 +7,14 @@
 *.dll
 *.so
 *.dylib
+build/
 
 # Test binary, built with `go test -c`
 *.test
 
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Dependency directories (remove the comment below to include it)
-# vendor/
```

Comment:
vendor/ is osolete way to manage dependencies (almost everyone migrated to go.mod), we don't use it